models/demos/deepseek_v3/tests/test_decoder_block.py

```diff
@@ -0,0 +1,236 @@
+# SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC.
+# SPDX-License-Identifier: Apache-2.0
+
+
+from random import randint
+
+import pytest
+import torch
+from loguru import logger
+
+import ttnn
+from models.demos.deepseek_v3.reference.modeling_deepseek import DeepseekV3DecoderLayer
+from models.demos.deepseek_v3.tt.decoder_block.decoder_block import DecoderBlock
+from models.demos.deepseek_v3.tt.mla_1d import MLA1D
+from models.demos.deepseek_v3.tt.rope import RotarySetup
+from models.demos.deepseek_v3.utils.reference_forwards import reference_forward_decode as reference_forward
+from models.demos.deepseek_v3.utils.run_config import create_run_config
+from models.demos.deepseek_v3.utils.test_utils import load_reference_io_tensors_for_module, load_state_dict
+from models.utility_functions import comp_pcc
+
+MAX_START_POS = 512
+
+
+@pytest.fixture
+def hf_config(hf_config):
+    """Load DeepSeek config for testing."""
+    hf_config.max_seq_len = 5 * 1024  # Set max sequence length for testing
```

Comment:
Which module uses this? Shouldn't we be testing with the actual max sequence length?